packages/node-cli/src/CLI.js

```diff
@@ -42,6 +42,11 @@ module.exports = class CLI {
 			const commandMetadataList = this._commandsMetadataService.getCommandsMetadata();
 			this._initializeCommands(commandMetadataList, runInInteractiveMode);
 
+			// If there are no arguments, we print SuiteCloud version header
+			if (!process.argv || !process.argv.slice(2).length) {
```

Comment:
Why do we need to slice? Can't we check (!process.argv.length || process.argv.length === 0) ?